databallpy/load_data/tracking_data/_add_player_tracking_data_to_dict.py

```diff
@@ -16,13 +16,22 @@ def _add_player_tracking_data_to_dict(
 
     Returns:
         dict: contains all tracking data
+
+    Note:
```

Comment:
Just to check whether I understand this:
It is faster to use try except because the try will only fail once per player.
While previously the if statement was checked for every frame.